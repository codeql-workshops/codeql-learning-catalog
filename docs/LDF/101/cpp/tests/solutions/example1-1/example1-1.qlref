example1-1.ql