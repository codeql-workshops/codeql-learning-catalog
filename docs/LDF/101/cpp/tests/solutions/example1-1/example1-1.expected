| file://:0:0:0:0 | operator= | operator= |
| file://:0:0:0:0 | operator= | operator= |
| main.cpp:4:6:4:6 | f | f |
